code/modules/clothing/under/accessories/holster.dm

```diff
@@ -182,8 +184,6 @@
 /obj/item/clothing/accessory/holster/utility
 	name = "utility holster"
 	desc = "A utility holster."
-	icon = 'icons/contained_items/clothing/accessories/utility_holsters.dmi'
```

Comment:
keep this as-is. the utility holsters have minor tweaks to the sprite.

Reply:
correct me if i'm wrong, but there is no difference between the brown holsters and the utility holsters whatsoever.

Reply:
there is.